services/karmaService.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const sqlite3 = require('sqlite3');
+const config = require('../config');
+const db = new sqlite3.Database(config.dbFilePath);
+
+function _getUserKarma(username, callback, message) {
+  db.each("SELECT SUM(point) as totalKarma FROM karma WHERE username = ?", username,
+    function(err, row) {
+      return callback(message + row.totalKarma);
+  });
+}
+
+module.exports = {
+  getKarma: (callback) => {
```

Comment:
Where is this being used ?
And it doesn't seem to be doing anything apart from `true` ?